base/base.go

```diff
@@ -325,3 +330,10 @@ func (s *Server) handleBotLogs(msg chat1.MsgSummary) error {
 	s.ChatEcho(msg.ConvID, "log output: %s", destFilePath)
 	return nil
 }
+
+func (s *Server) handleFeedback(msg chat1.MsgSummary) error {
+	s.Report("Feedback from @%s\n ```%s```", msg.Sender.Username, msg.Content.Text.Body)
```

Comment:
Might be nice to strip the `!feedback-prefix` from the message, but not necessary